lib/kitchen/directions/bake_figure.rb

```diff
@@ -11,16 +11,16 @@ def self.v1(figure:, number:, cases: false)
         figure.target_label(label_text: 'figure', custom_content: number, cases: cases)
 
         title = figure.title&.cut
-        caption = figure.caption&.cut
+        caption = figure.caption&.wrap_children('span', class: 'os-caption')&.cut
```

Comment:
It is a workaround for the above below usage of nokogori `children` method in caption baking (l.23). If the actual children are wrapped inside another element in this case `span.os-caption`, the `children` method during parsing caption children doesn't add additional new lines between the nested children elements.